examples/align/align_marker_on_segment.py

```diff
@@ -0,0 +1,94 @@
+import biorbd
+
+from biorbd_optim import (
+    Instant,
+    Axe,
+    OptimalControlProgram,
+    ProblemType,
+    Objective,
+    Constraint,
+    Bounds,
+    QAndQDotBounds,
+    InitialConditions,
+    ShowResult,
+    OdeSolver,
+)
+
+
+def prepare_ocp(biorbd_model_path="cube_and_line.bioMod", show_online_optim=False, ode_solver=OdeSolver.RK):
+    # --- Options --- #
+    # Model path
+    biorbd_model = biorbd.Model(biorbd_model_path)
+
+    # Problem parameters
+    number_shooting_points = 8
+    final_time = 0.5
```

Comment:
Please copy was is now done in Align_marker, so the number_shooting_points and final time is called from outside. So the example below can call it with larger values than the test